python_scripts/02_numerical_pipeline_scaling.py

```diff
@@ -390,9 +390,13 @@
 # trained to the full training set, along with an estimate of the variability
 # (uncertainty on the generalization accuracy).
 #
-# You can pass additional parameters to `cross_validate` to get more
-# information, for instance training scores. These features will be covered in
-# a future notebook.
+# Setting `cv=5` or `cv=10` is a common practice, as it is a good trade-off between
+# computation time and stability of the estimated variability.
```

Comment:
It might be good to mention here that in the case we would like to obtain a distribution estimation of the scores then we usually would repeat this 5- or 10-fold cross-validation. This cross-validation is strategy is known as `RepeatedKFold` in scikit-learn and could be provided to the `cv` parameter.

Reply:
I guess it would be more appropriate to mention `RepeatedKFold` in [this notebook](https://inria.github.io/scikit-learn-mooc/python_scripts/cross_validation_train_test.html), as we introduce here the concept of score distributions. This could be a good occasion to extend the discussion on such a topic, what do you think?